public/plugins/testdata/tsconfig.json

```diff
@@ -3,5 +3,6 @@
   "compilerOptions": {
     "typeRoots": []
   },
-  "include": ["./src", "./types", "../../app/types/*.d.ts", "../../../packages/grafana-ui/src/types/*.d.ts"]
+  "include": ["./src", ".config/types"],
```

Comment:
removed files out of the directory because now it was returning an error:
```
'rootDir' is expected to contain all source files.
  The file is in the program because:
    Matched by include pattern '../../app/types' in 'grafana/public/plugins/testdata/tsconfig.json'

  tsconfig.json:6:41
    6   "include": ["./src", ".config/types", "../../app/types", "../../../packages/grafana-ui/src/types/*.d.ts"],
                                              ~~~~~~~~~~~~~~~~~
    File is matched by include pattern specified here.

```